__tests__/sass/mdn/_button-states-mixins-test.scss

```diff
@@ -0,0 +1,16 @@
+@include describe('active-button-state') {
+    @include it(
+        'Returns an active button style that is applied to elements that are interactive and has focus'
+    ) {
+        @include assert {
+            @include output {
+                @include active-button-state();
+            }
+
+            @include expect {
+                margin: 1px 0 0 1px;
```

Comment:
I'm not a designer by any means, but I didn't love the little wobble it does on hover. But again, not a designer :).